net/unbound/Makefile

```diff
@@ -45,6 +45,16 @@ define Package/unbound/description
   This package contains the Unbound daemon.
 endef
 
+define Package/unbound/postinst
+#!/bin/sh
+
+group_exists unbound || group_add_next unbound
+
+gid=$$(grep -s '^unbound:' "$${IPKG_INSTROOT}/etc/group" | cut -d: -f3)
+
+user_exists unbound || user_add unbound '' "$$gid"
```

Comment:
We can define `USERID:=unbound=[uid]:unbound=[gid]` in `Package/unbound` and let the `default_postinst` do the group/user preparation for us.